daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -990,12 +990,8 @@ private[lf] object SBuiltin {
   /** $beginExercise
     *    :: arg                                           0 (choice argument)
     *    -> ContractId arg                                1 (contract to exercise)
-    *    -> List Party                                    2 (actors)
```

Comment:
Note: This change is just fixing a related comment which was just wrong/out-of-date.